packages/ag-charts-community/src/chart/annotation/annotationManager.ts

```diff
@@ -1,11 +1,57 @@
 import type { AgAnnotationsThemeableOptions } from '../../options/chart/annotationsOptions';
 import type { Group } from '../../scene/group';
 import type { Node } from '../../scene/node';
+import { isPlainObject } from '../../util/type-guards';
+import { BaseManager } from '../baseManager';
+import type { Memento, MementoOriginator } from '../memento';
 
-export class AnnotationManager {
+export interface AnnotationsRestoreEvent {
+    type: 'restore-annotations';
+    annotations: AnnotationsMemento['annotations'];
+}
+
+export class AnnotationsMemento implements Memento {
+    type = 'annotations';
+    version = 10;
```

Comment:
My thinking was that if there is a change in v11, this would change to:

```ts
class AnnotationsMemento implements Memento {
    version = 11;
}

class AnnotationsMemento10 implements Memento {
    version = 10;
}
```

So as to be precise about the version it is for. But it may prove to be unnecessary to have different classes when we do migrations.

Either way, when the memento is created, it is overwritten to be the version passed through from the caretaker.